src/components/ModalExport.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+import { Modal } from './Modal.js'
+import { DropDownMenu } from './DropDownMenu.js'
+import ArrowDown from '../images/keyboard_arrow_down_352466.svg'
+//  util's.js
+import {
+  ellipsize,
+  headValue,
+  getDescendants
+} from '../util'
+
+export const ModalExport = () => {
+  const dispatch = useDispatch()
+  const cursor = useSelector(state => state.cursor)
+  const execFunc = useSelector(state => state.execFunc)
+  const [format, setFormat] = React.useState({ f1: 'plaintext', f2: 'Plain Text' })
+  const [isOpen, handleMenu] = React.useState(false)
+  const exportInfo = `${ellipsize(headValue(cursor))} and ${getDescendants(cursor).length} subthoughts as ${format.f2} `
```

Comment:
- 'Export THOUGHT and...' should have quotes around the thought: 'Export "THOUGHT" and...'
- "1 subthoughts" should say "1 subthought"